recipe/meta.yaml

```
@@ -57,6 +55,9 @@ outputs:
         - itkTestDriver -- echo "Runtime ITK OK"
         - itkTestDriver --no-process --compare-MD5 ITK-JPEG.jpg 959e1682d19026fc62cfbdc140df31da
   - name: libitk-devel
+    build:
+      run_exports:
+        - {{ pin_subpackage('libitk', max_pin='x.x') }}
```

I think you also need a min_pin:

```suggestion
        - {{ pin_subpackage('libitk', min_pin='x.x', max_pin='x.x') }}
```

---

no it takes it implicitly from the current package version.